src/main/kotlin/lotto/domain/Lotto.kt

```diff
@@ -0,0 +1,27 @@
+package lotto.domain
+
+class Lotto(
+    private val numList: List<LottoNumber>
```

Comment:
사소한 부분이긴 하지만, List가 특별한 도메인을 나타내는게 아닌 이상 변수명은 복수형으로 나타내면 좋을 것 같아요.
```suggestion
    private val lottoNumbers: List<LottoNumber>
```